Autopilot/PathManager/Inc/cruisingState.hpp

```diff
@@ -10,6 +10,8 @@
 #ifndef CRUISING_STATE
 #define CRUISING_STATE
 
+constexpr int CRUISING_AIRSPEED {30};
```

Comment:
it was mentioned multiple times that this should be something like 15 m/s, no ? I'm not even sure Spike can achieve 30

Reply:
I'll test some other speeds out